### PR TITLE
Add zsh completion for 'docker {node,service,stack} ps --format'

### PR DESCRIPTION
**Pull request by sdurrheimer**
ref. #28213

Signed-off-by: Steve Durrheimer <s.durrheimer@gmail.com>

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM :cow:

File: contrib/completion/zsh/_docker [hunk 1]
```diff
@@ -1494,8 +1494,10 @@ __docker_node_subcommand() {
                 $opts_help \
                 "($help -a --all)"{-a,--all}"[Display all instances]" \
                 "($help)*"{-f=,--filter=}"[Provide filter values]:filter:->filter-options" \
+                "($help)--format=[Format the output using the given go template]:template: " \
```
Comment (AkihiroSuda) [inline contrib/completion/zsh/_docker]: Is this extra space intentional?

Comment (sdurrheimer) [inline contrib/completion/zsh/_docker]: yes, just a little convention in the script when an option needs a argument but we don't provide values.

**Comment by sdurrheimer** [comment]
Rebased

**Comment by sdurrheimer** [comment]
Rebased.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**